updater/updater.go

```diff
@@ -93,23 +93,16 @@ func Run(interval time.Duration, st *utils.Stopper) {
 					doneC <- true
 				}()
 
-				// Refresh the lock until the update is done.
 				for done := false; !done; {
 					select {
 					case <-doneC:
 						done = true
 					case <-time.After(refreshLockDuration):
+						// Refresh the lock until the update is done.
 						database.Lock(flagName, lockDuration, whoAmI)
 					}
 				}
 
-				// Write the last update time to the database and set the next update
-				// time.
-				now := time.Now().UTC()
-				database.UpdateFlag(flagName, strconv.FormatInt(now.Unix(), 10))
-				healthLatestSuccessfulUpdate = now
-				nextUpdate = now.Add(interval)
```

Comment:
If we remove that line, an updater that failed will just wait some random seconds and retry. I guess that's ok?